src/main/webapp/adminOnly/AdminOnly.js

```
@@ -48,43 +51,44 @@ export const AdminOnly = hh(class AdminOnly extends Component {
-        formData.bioMedical = issue.data.extraProperties.bioMedical;
-        formData.irbExpirationDate = issue.data.extraProperties.irbExpirationDate;
-        formData.projectStatus = issue.data.extraProperties.projectStatus;
-        initial = createObjectCopy(formData);
+    Project.getProject(this.props.projectKey).then(issue => {
+      let formData = {};
+      let initial = {};
+      this.props.initStatusBoxInfo(issue.data);
+      formData.projectKey = this.props.projectKey;
+      formData.investigatorFirstName = issue.data.extraProperties.investigatorFirstName;
+      formData.investigatorLastName = issue.data.extraProperties.investigatorLastName;
+      formData.degrees = issue.data.extraProperties.degrees;
+      formData.preferredIrb = isEmpty(issue.data.extraProperties.irb) ? '' : JSON.parse(issue.data.extraProperties.irb);
+      formData.preferredIrbText = issue.data.extraProperties.irbReferralText;
+      formData.trackingNumber = issue.data.extraProperties.protocol;
+      formData.projectTitle = issue.data.extraProperties.projectTitle;
+      formData.initialDate = issue.data.extraProperties.initialDate;
+      formData.sponsor = this.getSponsorArray(issue.data.fundings);
+      formData.initialReviewType = isEmpty(issue.data.extraProperties.initialReviewType) ? '' : JSON.parse(issue.data.extraProperties.initialReviewType);
+      formData.bioMedical = issue.data.extraProperties.bioMedical;
+      formData.irbExpirationDate = issue.data.extraProperties.irbExpirationDate;
+      formData.projectStatus = issue.data.extraProperties.projectStatus;
+      initial = createObjectCopy(formData);
+      if (this._isMounted) {
         this.setState(prev => {
           prev.formData = formData;
           prev.initial = initial;
           prev.isAdmin = component.isAdmin;
           return prev;
         })
-      }).catch(error => {
-        this.setState(() => { throw error; });
-      });
+      }
+    }).catch(() => { });
```

A catch-all block without a logger isn't a good idea. Please add logging here.